packages/dapp-svelte-wallet/api/src/lib-wallet.js

```diff
@@ -108,18 +144,24 @@ export async function makeWallet({
   const idToOutcome = new Map();
 
   // Client-side representation of the purses inbox;
+  /** @type {Map<string, PursesJSONState>} */
   const pursesState = new Map();
+
+  /** @type {Map<Purse, PursesFullState>} */
   const pursesFullState = new Map();
   const inboxState = new Map();
 
-  // The default Zoe invite purse is used to make an offer.
+  /**
+   * The default Zoe invite purse is used to make an offer.
+   * @type {Purse}
+   */
   let zoeInvitePurse;
 
   function getSortedValues(map) {
     const entries = [...map.entries()];
     // Sort for determinism.
     const values = entries
-      .sort(([id1], [id2]) => id1 > id2)
+      .sort(([id1], [id2]) => cmp(id1, id2))
```

Comment:
This is more than a typing change. It is a bug fix. Any other semantically significant changes I should notice?

Reply:
You caught the two.  These were changes necessary to make the types work, so I bundled them with the typings.  Next time I do this, I will make separate steps.

Reply:
That would be fine. Also fine is just calling my attention to them so I don't miss them. Thanks.